linux

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
```

Comment:
can we add a way to exclude some tools
eg. If I don't want to install ngrok then, `bash script.sh -e ngrok`

Reply:
@ShubhamRasal  we can comment out unwanted install. -e will make script more complex.